src/core/types/addons.js

```diff
@@ -47,6 +47,12 @@ export type AddonAuthorType = {|
   username: string,
 |};
 
+type LanguageToolType = {|
```

Comment:
I find this name misleading since it implies it's the full type of a language tool add-on. Instead, it's used here to indicate the extra fields possessed by language tools. I think that's fine given some other limitations we have for expressing types in this file. I would just recommend changing the name to `LanguageToolData` which follows the pattern we have for things like `ThemeData` within this file.

Reply:
Makes sense, thanks.